src/signals/incident/components/form/HandlingMessage/__snapshots__/index.test.js.snap

```diff
@@ -2,43 +2,35 @@
 
 exports[`Form component <HandlingMessage /> rendering should render empty correctly with no handling message 1`] = `
 <div
-  className="col-12 mode_input"
+  className="col-12"
```

Comment:
Can we delete this snap and replace the tests with testing-library?

Reply:
Fixed it :) now in TypeScript and without the snapshots (using testing-library)